src/MultiplicationTable.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class MultiplicationTable {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int amountColumns = scanner.nextInt();
+        int amountLines = scanner.nextInt();
+        for (int i = 1; i <= amountColumns; i++) {
+            for (int k = 1; k <= amountLines; k++) {
```

Comment:
Please, use `j` instead of `k`
Or be more descriptive and use `row` and `column`

Reply:
Hello?